pkgs/by-name/go/golem/package.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  protobuf,
+  redis,
+  fontconfig,
+  versionCheckHook,
+  nix-update-script,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "golem";
+  version = "1.0.23";
+
+  src = fetchFromGitHub {
+    owner = "golemcloud";
+    repo = "golem";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-ZvqlWavntlzQLUuTZG+pCeEZTT+DTP+QjiRgXu/9TvQ=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    protobuf
+  ];
+
+  buildInputs = [
+    fontconfig
+    (lib.getDev openssl)
+  ];
+
+  env.VERSION = version;
+
+  # Required for golem-wasm-rpc's build.rs to find the required protobuf files
+  # https://github.com/golemcloud/wasm-rpc/blob/v1.0.6/wasm-rpc/build.rs#L7
+  GOLEM_WASM_AST_ROOT = "/build/cargo-vendor-dir/golem-wasm-ast-1.0.1";
```

Comment:
Can we use relative paths here? Otherwise it probably just fails on Darwin because iirc the prefix there is different.

Reply:
using relative paths now